docs/guides/hedgehog-engine/unleashed/levels/glvl.md

```diff
@@ -0,0 +1,59 @@
+---
+description: Sonic GLvl setup tutorial for Sonic Unleashed
+---
+
+# How to use Sonic GLvl with Sonic Unleashed
+
+!!! note
+    This guide covers how to setup Sonic Unleashed levels in GLvl, but does not cover how to use this tool. It's recommended that you also read the [Sonic Generations GLvl guide](/guides/hedgehog-engine/blueblur/levels/glvl.md).
+
+!!! info
+    This guide uses the following tools:
+
+    - [Sonic GLvl](/tools/hedgehog-engine/blueblur/levels/){ target="_blank"}
+    - [Unleashed Mod Toolbox](/tools/hedgehog-engine/unleashed/files){ target="_blank"}  
+    - [Unleashed Object Template](assets/glvl/UnleashedObjectTemplate.zip)
+    - Xbox 360 SDK, find this yourself
+
+#### Preparing .ar Files
+This example will be editing Windmill Isle Act 1 Day. Open your extracted Unleashed game folder and locate your files. In my case the files are 
+````
+#ActD_MykonosAct1.ar.00
+#ActD_MykonosAct1.arl
+ActD_MykonosAct1.ar.00 
+ActD_MykonosAct1.ar.01
+ActD_MykonosAct1.arl
+````
+Once located you will need to copy those files to the #files folder inside your Mod Toolbox directory. now open #SUModToolbox.bat and press 3 and enter to decompress the files, press 1 or 2 depending on the system and press enter.
```

Comment:
Please add a blank line after the code block. If you don't add it, then this text will have a larger size on the page, which isn't intended.